aiida/work/legacy/job_process.py

```diff
@@ -151,3 +151,9 @@ def _setup_db_record(self):
 
         if parent_calc:
             self._calc.add_link_from(parent_calc, "CALL", LinkType.CALL)
+
+        if self.raw_inputs:
```

Comment:
Maybe, but wouldn't the same issue be hit at the same place in `Process._setup_db_record`? That uses the exact same code (lines 377-381). If so, we should also fix it there I guess